### PR TITLE
Louis XIII

### PR DESCRIPTION
**Pull request by Paradact**
I have added the man that was thirteenth

### CODE REVIEW

File: consts.js [hunk 1]
```diff
@@ -25,13 +25,14 @@ var thirteenStrings = [
     "PT", // PT is Brazilian political party represented by the number 13
     "Washington Luís", // Brazil's thirteenth president
     "Millard Fillmore", // Thirteenth President of the United States
+    "Louis XIII" // Thirteenth king of France
```
Comment (Almenon) [inline consts.js]: Trajan was a pretty awesome dude, but why Louis? What good things did he do?

Comment (unimatrixZxero) [reply consts.js]: He was gay and made men wearing wigs fashionable?

Comment (Almenon) [reply consts.js]: good point.

Comment (Almenon) [inline consts.js]:
```suggestion
    "Louis XIII", // Thirteenth king of France
```

Don't forget the comma! :)

**Comment by Almenon** [comment]
Long live the king!

**Pull request merged**